### PR TITLE
Rewrite exists(property) to property IS NOT NULL. No AUTO cherry-pick

### PR DESCRIPTION
**Pull request by Lojjs**
Update APOC syntax to adapt to removal of exists(property) in Neo4j.

### CODE REVIEW

File: core/src/test/java/apoc/schema/SchemasTest.java [hunk 1]
```diff
@@ -551,7 +554,7 @@ public void testIndexesMoreLabels() {
     @Test
     public void testSchemaRelationshipsExclude() {
         ignoreException(() -> {
-            db.executeTransactionally("CREATE CONSTRAINT FOR ()-[like:LIKED]-() REQUIRE exists(like.day)");
+            db.executeTransactionally("CREATE CONSTRAINT FOR ()-[like:LIKED]-() REQUIRE like.day I");
```
Comment (Hunterness) [inline core/src/test/java/apoc/schema/SchemasTest.java:557]: I feel like some part is missing here?

**Review by Hunterness — CHANGES_REQUESTED** [review]

**Review by Hunterness — APPROVED** [review]

File: docs/asciidoc/modules/ROOT/partials/usage/apoc.schema.relationships.adoc [hunk 2]
```diff
@@ -16,6 +16,6 @@ CALL apoc.schema.relationships();
 .Results
 [opts="header"]
 |===
-| name                                                         | type                              | properties | status
-| "CONSTRAINT ON ()-[liked:LIKED]-() ASSERT exists(liked.day)" | "RELATIONSHIP_PROPERTY_EXISTENCE" | ["day"]    | ""
+| name                                                             | type                              | properties | status
+| "CONSTRAINT ON ()-[liked:LIKED]-() ASSERT liked.day IS NOT NULL" | "RELATIONSHIP_PROPERTY_EXISTENCE" | ["day"]    | ""
```
Comment (Hunterness) [inline docs/asciidoc/modules/ROOT/partials/usage/apoc.schema.relationships.adoc:20]: Actually this should be updated to use `FOR .. REQUIRE` as well

**Review by Hunterness — CHANGES_REQUESTED** [review]

File: docs/asciidoc/modules/ROOT/partials/usage/apoc.schema.relationships.adoc (hunk 2 above)
Comment (Lojjs) [reply docs/asciidoc/modules/ROOT/partials/usage/apoc.schema.relationships.adoc:20]: Yes. I get over 100 matches on `CONSTRAINT ON` on apoc dev though so I think we need a separate PR for that though if you don't have one open already?

**Review by AzuObs — APPROVED** [review]

File: docs/asciidoc/modules/ROOT/partials/usage/apoc.schema.relationships.adoc (hunk 2 above)
Comment (Hunterness) [reply docs/asciidoc/modules/ROOT/partials/usage/apoc.schema.relationships.adoc:20]: Sure lets do it separately then :)

**Pull request merged**